test/sym/memory.wast

```diff
@@ -61,6 +62,12 @@
   )
 
   (func $start
+    i32.const 1
+    memory.grow
```

Comment:
I believe this is not needed because `(memory $m 1)` already gives a size 1 to the memory as it's the minimal size. :-)

Reply:
And there's no symbolic value in this test. :-)

Reply:
I've tried a `memory.grow` with a symbolic value :
```console
owi: internal error, uncaught exception:
  Failure("Unsupported symbolic value reasoning over \"(i32.wrap_i64 (i64.mul (i64.extend_i32_s symbol_0) (i64 65536)))\"")
```
What type of error is it?

Reply:
Can i perform a memory load or store at a symbolic position ?

Reply:
> What type of error is it?

It's related to your second question:

Currently no, but I just started working on it. I'll opt to concretize addresses for now. I'll to submit a PR today :smiley:

Reply:
```quote
I've tried a memory.grow with a symbolic value :
```

Yes, I know it's not working with a symbolic value, but what I meant is that the test is useless without the symbolic value (because we are already testing this) so you can basically remove the test for now.